Formula/b/bozohttpd.rb

```diff
@@ -0,0 +1,41 @@
+class Bozohttpd < Formula
+  desc "Small and secure http version 1.1 server"
+  homepage "http://www.eterna.com.au/bozohttpd/"
+  url "http://www.eterna.com.au/bozohttpd/bozohttpd-20220517.tar.bz2"
+  sha256 "9bfd0942a0876e5529b0d962ddbcf50473bcf84cf5e4447043e4a0f4ea65597a"
+  license "BSD-2-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "lua"
+  depends_on "openssl@3"
+
+  def install
+    cflags = [
+      "-DNO_BLOCKLIST_SUPPORT",
+      "-DDO_HTPASSWD",
+    ]
```

Comment:
```suggestion
    # some comment, ideally with a link, explaining what these are doing, how you figured that out and where to check for stuff like this in future
    cflags = [
      "-DNO_BLOCKLIST_SUPPORT",
      "-DDO_HTPASSWD",
    ]
```

Reply:
* Did add some comments explaining both flags
  * Removed the `ensure`... that one is indeed not needed. `bozohtppd` process  goes away after test